ext/dcos-installer/dcos_installer/installer_analytics.py

```diff
@@ -2,33 +2,36 @@
 import os
 import uuid
 
+from dcos_installer import backend
+
 
 class InstallerAnalytics():
-    def __init__(self, config):
+    def __init__(self):
         self.uuid = self.new_uuid()
-        self.bootstrap_version = os.getenv("BOOTSTRAP_ID")
-        self.customer_key = config["customer_key"] if 'customer_key' in config else ""
+        self.bootstrap_version = os.getenv("BOOTSTRAP_ID", "")
+        self.customer_key = ""
```

Comment:
I don't see value in having this member. We always just get the customer_key out of the configuration (https://github.com/dcos/dcos/pull/43/files?diff=split#diff-2067a16f4c753c685193f0c92a648ea8R24), so we never actually use this member's value, all it could be is old / outdated.